lib/python-caller-trampoline.py

```diff
@@ -69,8 +69,17 @@ def try_dumps(obj, sort_keys=False, allow_nan=False):
 
         # check whether we have the desired fcn in the module
         if hasattr(mod, fcn):
-            # call the desired function in the loaded module
+            # get the desired function in the loaded module
             method = getattr(mod, fcn)
+
+            # check if the desired function is a legacy element function - if so,
+            # we assume that ... and add argument for element_index
+            sig = signature(method)
+            if len(sig.parameters) == 3:
+                if all(p in sig.parameters for p in ['element_html', 'element_index', 'data']):
```

Comment:
nitpick: it would be slightly more specific to do `if sig.parameters[0] == 'element_html' and sig.parameters[1] == 'element_index' and sig.parameters[2] == 'data':`

Reply:
I agree. Fixed.